common/autoware_testing/design/autoware_testing-design.md

```diff
@@ -51,6 +54,52 @@ build/<package_name>/test_results/<package_name>/<executable_name>_smoke_test.xu
 
 ```
 
+### Interface test
+
+To add a interface test to your package tests, add test dependency on `autoware_testing` to `package.xml`
+
+```{xml}
+<test_depend>autoware_testing</test_depend>
+```
```

Comment:
Please remove `{}` because it's not highlighted.

![image](https://user-images.githubusercontent.com/31987104/154266409-c1fc3844-214e-4d2f-bcfc-6c8aefa906a4.png)

Reply:
Okay. Updated 1208f2acc2c53727a0821e41dfb730e6eeb65bb5